spec/octocatalog-diff/integration/preserve_environments_spec.rb

```diff
@@ -40,20 +40,16 @@
             '-n', 'rspec-node.github.net',
             '--environment', 'asdfgh',
             '--to-catalog', OctocatalogDiff::Spec.fixture_path('catalogs/default-catalog-v4.json'),
-            '--hiera-config', 'environments/production/config/hiera.yaml',
+            '--hiera-config', 'config/hiera.yaml',
             '--hiera-path-strip', '/var/lib/puppet', '--no-parallel'
           ]
         )
       end
 
       it 'should exit without error' do
-        expect(@result.exitcode).to eq(0), OctocatalogDiff::Integration.format_exception(@result)
+        expect(@result.exitcode).to eq(2), OctocatalogDiff::Integration.format_exception(@result)
```

Comment:
This one should be reverted as per my earlier comment - change the 2 back to a 0 here.